prov/rxd/src/rxd_cq.c

```diff
@@ -1064,6 +1065,7 @@ static void rxd_handle_start_data(struct rxd_ep *ep, struct rxd_peer *peer,
 repost:
 	rxd_ep_repost_buff(rx_buf);
 out:
+	assert(rxd_reposted_bufs);
```

Comment:
in some cases buffer should not be re-posted (in case if unexpected packet received), in this case we should be sure that there are at least one preposted buffer... I'm not sure what is the best solution here, but in any case zero-posted buffers produce deadlock